src/App.js

```diff
@@ -12,7 +12,7 @@ class App extends Component {
           <div className="Logo-header">
             <img src={headerBackground} className="App-logo-back" alt="logo-back" />
             <img src={logo} className="App-logo" alt="logo" />
-	  </div>
+	        </div>
```

Comment:
Fix this unaligned extra white-space?...

Reply:
@TheCodeArtist for to push to changes .... done in latest commit.